insights/specs.py

```diff
@@ -331,10 +331,12 @@ def docker_container_ids(broker):
 md5chk_files = sf.simple_command("/bin/ls -H /usr/lib*/{libfreeblpriv3.so,libsoftokn3.so} /etc/pki/product*/69.pem /etc/fonts/fonts.conf /dev/null 2>/dev/null", name="md5chk_files")
 prelink_orig_md5 = None
 prev_uploader_log = sf.simple_file("var/log/redhat-access-insights/redhat-access-insights.log.1", name="prev_uploader_log")
-ps_aux = sf.simple_command("/bin/ps aux", name="ps_aux")
-ps_auxcww = sf.simple_command("/bin/ps auxcww", name="ps_auxcww")
-ps_auxwww = sf.simple_file("sos_commands/process/ps_auxwww", name="ps_auxwww", context=HostArchiveContext)
-ps_axcwwo = sf.simple_command("/bin/ps axcwwo ucomm,%cpu,lstart", name="ps_axcwwo")
+ps_auxww = sf.first_of([
+    sf.simple_command("/bin/ps auxww", name="ps_auxww"),
+    sf.simple_file('sos_commands/process/ps_aux'),
```

Comment:
I'm not sure if it's OK to use `simple_command` instead of the `simple_file` here. In 1.x, e.g. the file "sos_commands/process/ps_aux" can also be matched by `CommandSpec("/bin/ps aux")`. And for `simpe_file`, better mark it as `context=HostArchiveContext`.
I think we need @csams to look over this.

Reply:
OK, I've marked them as `HostArchiveContext` files - I think that probably solves the problem there.  We don't want it to actually try to execute any of the other commands - they're just there to pick up input from sosreports or previous archives.